scripts/config/msu/classes/empty_bb_class.nut

```diff
@@ -0,0 +1,6 @@
+this.empty_bb_class <- {
+	m = {},
+	function create()
+	{
+	}
+}
```

Comment:
I think this should be in a different folder, maybe bbclasses. I don't think it should be grouped together with Squirrel classes